site/content/guides/grpc.md

```diff
@@ -145,10 +145,44 @@ spec:
 
 ## Gateway API Configuration
 
-At the moment, configuring gRPC routes with Gateway API resources is achieved by the same method as with Ingress v1: annotation to select a protocol and port on a Service referenced by HTTPRoute `spec.rules[].backendRefs`.
+Gateway API now supports a specific resource [GRPCRoute][6] for routing gRPC requests.
```

Comment:
Please move this content into `site/content/docs/main/guides/grpc.md` instead, thanks!

Reply:
in main branch, there is no folder site/content/docs/main/guides/, shall I create one and git mv site/content/guides/grpc.md to there?

Reply:
https://github.com/projectcontour/contour/tree/main/site/content/docs/main/guides